ui/app/components/sentinel-policy-editor.hbs

```diff
@@ -56,21 +56,21 @@
 		<G.Radio::Field
 			@id="advisory"
 			checked={{eq @policy.enforcementLevel "advisory"}}
-			data-test-token-type="client"
+			data-test-enforcement-level="advisory"
 		as |F|>
 			<F.Label>Advisory</F.Label>
 		</G.Radio::Field>
 		<G.Radio::Field
 			@id="soft-mandatory"
 			checked={{eq @policy.enforcementLevel "soft-mandatory"}}
-			data-test-token-type="soft-mandatory"
+			data-test-enforcement-level="soft-mandatory"
 		as |F|>
 			<F.Label>Soft Mandatory</F.Label>
 		</G.Radio::Field>
 		<G.Radio::Field
 			@id="hard-mandatory"
 			checked={{eq @policy.enforcementLevel "hard-mandatory"}}
-			data-test-token-type="hard-mandatory"
+			data-test-enforcement-level="hard-mandatory"
```

Comment:
The original templates was copied from ACL policy/token templates; this fixes the test-layer-only attributes.

Note: attributes that begin with `data-test-` are automatically stripped out for non-test builds.